server/apps/main/templates/main/experiences_page.html

```diff
@@ -177,7 +177,7 @@ <h5 class="card-title story-header">{{ experience.number }}. {% firstof experien
           aria-expanded="false"
           aria-controls="single{{ experience.experience_id }}"
           style="text-align: right">
-          Read Story on Single Page
+          Read More
```

Comment:
This one might be a bit confusing, given that the other button for expanding the is called _Read Full Story_ and already shows the whole experience. Maybe we could do the following: 

1. Rename the _other_ button from _Read Full Story_ to _Expand_
2. Rename this button to _Open in new page_